poc/flp_pine.py

```diff
@@ -69,31 +69,29 @@ def __init__(self,
                              "number of fractional bits, that causes the "
                              "encoded norm bound to be larger than "
                              "field modulus.")
-        self.encoded_sq_norm_bound = self.Field(
```

Comment:
There should be some places in the draft that also has `encoded_sq_norm_bound`, can you change those too?